Elevel/Elevel.Api/Controllers/UserController.cs

```diff
@@ -25,5 +25,20 @@ public async Task<IActionResult> GetTokenAsync(TokenRequestModel model)
             var result = await _userService.GetTokenAsync(model);
             return Ok(result);
         }
+
+        [HttpGet("info")]
+        public async Task<IActionResult> UserInfo()
+        {
+            var claims = User.Claims.ToList();
+            var userId = claims.FirstOrDefault(x=>x.Type == "uid").Value;
```

Comment:
this can fail, because FirstOrDefault can return null and you will get NRE, I advise to use in such cases FIrst, it will throw more descriptive exception message.